tools/make.py

```diff
@@ -697,11 +697,9 @@ def version_stamp_pboprefix(module,commitID):
         f.close()
 
         if configtext:
-            patchestext = re.search(r"version.*?=.*?$", configtext, re.DOTALL)
-            if patchestext:
+            if re.search(r"version=(.*?)$", configtext, re.DOTALL):
```

Comment:
You are right, my bad. Anyways, same comment as below and this is not used anyways. Thanks for correcting me!